difflint/lint.py

```diff
@@ -1,20 +1,43 @@
 # Copyright 2015 Endless Mobile, Inc.
 
+import functools
 import json
 import os.path
+import pathlib
 import pep8
 from pkg_resources import resource_filename
 import pyflakes.api
 import shutil
+import sys
 
 from .lint_output import LintOutput
 from .python_reporters import PEP8TerseReporter, PyFlakesTerseReporter
+from .utils import repo_root
 
-_ENABLED_LINTERS_CONFIG = resource_filename(__name__, 'data/.difflintrc')
+@functools.lru_cache()
+def _get_enabled_linters_config_path():
+    """
+    Returns the path of the file describing which linters are enabled
+    based upon whether a custom, repository-specific file exists.
+
+    Inputs: None
+
+    Output: A patlib.Path object.
+    """
+    custom_path = repo_root() / '.difflintrc'
+    if custom_path.is_file():
+        return custom_path
+    else:
```

Comment:
Personally, I tend to find it more readable without the else, because it is redundant with the return. As with many things, I have met other engineers who feel differently.